lambda/app/src/routes.ts

```diff
@@ -305,7 +305,18 @@ export const setRoutes = (app: any) => {
       try {
         const { id } = req.params || {};
         const { start, end, env } = req.query || {};
-        const { status, message, data } = await fetchLogs(req.session, env, id, start, end);
+        const userRequest = await getAllowedRequest(req.session, id);
+        if (!userRequest) {
+          return res.status(403).send('forbidden');
+        }
+        const { status, message, data } = await fetchLogs(
+          env,
+          userRequest.clientId,
+          userRequest.id,
+          start,
+          end,
+          req.session.idir_userid,
```

Comment:
will all log's sessions have an idir_userid?

Reply:
yes in the app router the idir_userid is added to the session for all routes in the authentication middleware